backbone.js

```diff
@@ -70,6 +70,23 @@
 
   // Proxy Underscore methods to a Backbone class' prototype using a
   // particular attribute as the data argument
+  var modelMatcher = function(attrs) {
+    var matcher = _.matches(attrs);
+    return function(model) {
+      return matcher(model.attributes);
+    };
+  };
+  var modelProperty = function(key) {
+    return function(model) {
+      return model == null ? void 0 : model.get(key);
+    };
+  };
+  var cb = function(iteratee, instance) {
+    if (_.isFunction(iteratee)) return iteratee;
+    if (_.isObject(iteratee) && !instance._isModel(iteratee)) return modelMatcher(iteratee);
```

Comment:
Ok cool. We're probably fine but this would be a bug if we ever introduced 3- or 4-length methods in the future.